backend/controllers/appointments/addAppointment.js

```diff
@@ -124,8 +213,22 @@ const addAppointmentFactory = (appointmentsService, existingEventsService, elite
     if (errors.length > 0) {
       const { endOfPeriod } = endRecurringEndingDate({ date, startTime, repeats, times })
 
+      const offenderDetails = await elite2Api.getDetails(res.locals, offenderNo)
+      const formattedName = formatName(offenderDetails.firstName, offenderDetails.lastName)
+
+      const prisonerName =
```

Comment:
This needs to be centralised, possibley moving it into it's own nunjucks component.